src/lib.rs

```diff
@@ -177,6 +175,9 @@ impl Game {
             Some(c) => *c,
         };
 
+        let mut killed_player: Option<&mut Player> = None;
+        let closest_distance = self.kill_distance;
```

Comment:
Oh, clever way to both get the closest player _and_ restrict based on kill distance!

Reply:
:brain: